app/views/bookings/show.html.erb

```diff
@@ -0,0 +1,50 @@
+<%= render "common/header/menu" %>
+
+<div class="row">
+  <div class="col-md-6 col-offset-3 mx-auto confirmation-form">
+    <h4><%= t "booking.confirm_info_message" %></h4>
+    <div class="row mb-2">
+      <div class="col-md-6">
+        <div class="font-weight-bold d-inline"><%= t "tour.place" %></div>
+        <%= @tour_detail.tour.place %>
+      </div>
+      <div class="col-md-6">
+        <div class="font-weight-bold d-inline"><%= t "tour.price" %></div>
+        <%= t("tour_price", price: exchange_money(@tour_detail.price)) %>
+      </div>
+      <div class="col-md-6">
+        <div class="font-weight-bold d-inline"><%= t "tour.start_date" %></div>
+        <%= l(@tour_detail.start_date, format: :notime) %>
+      </div>
+      <div class="col-md-6">
+        <div class="font-weight-bold d-inline"><%= t "tour.end_date" %></div>
+        <%= l(@tour_detail.end_date, format: :notime) %>
+      </div>
+    </div>
+    <%= form_for @booking, class: "form-group" do |f| %>
+      <%= f.hidden_field :user_id, value: current_user.id %>
+      <%= f.hidden_field :tour_detail_id, value: @tour_detail.id %>
+      <%= f.label :number_of_bookers, t("booking.number_of_booker") %>
+      <%= f.select :number_of_bookers, options_for_select(1..5) %>
+      <%= f.hidden_field :current_price, value: @tour_detail.price %>
```

Comment:
Sao lại hidden field này

Reply:
Dạ em hidden field để truyền params đủ các trường trong table booking luôn

Reply:
Sao chú hok hiển thị lên luôn mà hidden ấy

giả sử  booking này đang sale, đúng giây cuối chị bấm order giá là 1 củ, sau khi nhập xong thôn tin nó lên giá lại 5 củ, thì chú hiển thị giá lên, chứ ri chú âm thâm insert 5 củ thì khách hàng nó chửi sấp mặt

Reply:
Dạ phía trên em có để thẻ khác để hiển thị toàn bộ thông tin rồi chị. Em nghĩ là lúc show form này ra thì giá trị của `price` đã lưu trong biến rồi nên khi gửi nó sẽ gửi giá trị 1 củ như ví dụ của chị. Trong form em hidden các trường kia để lúc truyền nó đẩy đi theo params, em chỉ hiển thị 2 trường là tên người đi với số điện thoại liên hệ người đi để user tiện chỉnh sửa. Với một lý do nữa là trong form các trường hidden sẽ dc mã hóa nên nó bảo mật tốt hơn.